gnews/gnews.py

```diff
@@ -152,15 +218,25 @@ def get_news(self, key):
         """
         if key:
             key = "%20".join(key.split(" "))
-            url = BASE_URL + '/search?q={}'.format(key) + self._ceid()
-            return self._get_news(url)
+            query = '/search?q={}'.format(key)
+            if self.end_date is not None:
```

Comment:
Instead of checking everywhere, is it possible we can check this in `_ceid` function? 

``` py
def _ceid(self):
       if self.end_date :
           query += "%20after%3A{}".format(self.start_date)
```